assets/javascripts/discourse/controllers/subscriptions.js

```diff
@@ -0,0 +1,37 @@
+import Controller from "@ember/controller";
+import Ember from "ember";
+import { htmlSafe } from "@ember/template";
+import I18n from "I18n";
+
+export default Controller.extend({
+  init() {
+    // Perform any initialization logic here
+    this._super(...arguments);
+    // Additional custom initialization code
+    if (this.currentUser) {
+      this.currentUser
+        .checkEmail()
+        .then(() => this.set("email", this.currentUser.email));
+    }
+  },
+  pricingTable: Ember.computed("email", function () {
```

Comment:
Instead of using computed properties you can use native getters in Ember.
https://guides.emberjs.com/release/upgrading/current-edition/tracked-properties/